### PR TITLE
APIServerTracing beta docs update

### PR DESCRIPTION
**Pull request by dashpole**
Enhancement issue: https://github.com/kubernetes/enhancements/issues/647

Kubernetes PRs:

* https://github.com/kubernetes/kubernetes/pull/113693
* https://github.com/kubernetes/kubernetes/pull/113698
* https://github.com/kubernetes/kubernetes/pull/113642
* https://github.com/kubernetes/kubernetes/pull/113545
* https://github.com/kubernetes/kubernetes/pull/113233
* https://github.com/kubernetes/kubernetes/pull/113217
* https://github.com/kubernetes/kubernetes/pull/113183
* https://github.com/kubernetes/kubernetes/pull/113172
* https://github.com/kubernetes/kubernetes/pull/112979
* https://github.com/kubernetes/kubernetes/pull/113682
* https://github.com/kubernetes/kubernetes/pull/113728

This is a placeholder PR.

### CODE REVIEW

File: content/en/docs/concepts/cluster-administration/system-traces.md [hunk 1]
```diff
@@ -59,22 +59,20 @@ as the kube-apiserver is often a public endpoint.
...
+apiVersion: apiserver.config.k8s.io/v1beta1
 kind: TracingConfiguration
 # default value
 #endpoint: localhost:4317
 samplingRatePerMillion: 100
 ```
 
 For more information about the `TracingConfiguration` struct, see
-[API server config API (v1alpha1)](/docs/reference/config-api/apiserver-config.v1alpha1/#apiserver-k8s-io-v1alpha1-TracingConfiguration).
+[API server config API (v1beta1)](/docs/reference/config-api/apiserver-config.v1alpha1/#apiserver-k8s-io-v1alpha1-TracingConfiguration).
```
Comment (sftim) [inline content/en/docs/concepts/cluster-administration/system-traces.md:75]: Is this right?
```suggestion
[API server config API (v1beta1)](/docs/reference/config-api/apiserver-config.v1beta1/#apiserver-k8s-io-v1beta1-TracingConfiguration).
```

Comment (dashpole) [reply content/en/docs/concepts/cluster-administration/system-traces.md:75]: I need to update the link.  I assume that API documentation is auto-generated?

Comment (sftim) [reply content/en/docs/concepts/cluster-administration/system-traces.md:75]: Yep, it is. https://kubernetes.io/docs/contribute/generate-ref-docs/ explains the process (which the release docs team do)

**Comment by dashpole** [comment]
The feature-gate flip had to be reverted.  APIServerTracing will not graduate this release.

**Pull request closed without merging**